cookiecutter.json

```diff
@@ -15,6 +15,7 @@
   "support_py2": "n",
   "use_pypi_deployment_with_travis": "y",
   "use_ghpages_deployment_with_travis": "y",
+  "integrate_codecov": "y",
```

Comment:
rename to `"use_codecov"` for consistency with the other similar options

Reply:
Fair point. Changing it!